src/Neo.VM/EvaluationStack.cs

```diff
@@ -69,7 +69,7 @@ IEnumerator IEnumerable.GetEnumerator()
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal void Insert(int index, StackItem item)
         {
-            if (index > innerList.Count) throw new InvalidOperationException($"Insert out of bounds: {index}/{innerList.Count}");
+            if (index > innerList.Count) throw new InvalidOperationException($"Insert index {index} is out of stack bounds: {index}/{innerList.Count}");
```

Comment:
This message contains duplicating information (`index`), I'd say it's excessive a bit.

Reply:
updated.